Ukrainian localization

DinoSlavik opened the pull request:
All but most of the manuals have been translated (only the home page and "basic" manuals from the "general" folder have been translated)




(A slightly modified version of the previous PR)

DinoSlavik committed:
Very-very basic ukrainian localization

Added ukrainian to pack.mcmeta
Almost completely translated uk_UA.lang

DinoSlavik committed:
Update uk_UA.lang

DinoSlavik committed:
Basic ukrainian docs

Translated basic docs and added a warning about translation inaccuracies to the main docs page (index.md)

DinoSlavik committed:
Fix

- Slightly improved items and blocks translation
- Changed file names to match the master branch (previously, for some reason, the file names were different...)

DinoSlavik committed:
MORE FILENAME FIXES FOR GOD OF FIXES

Yeah, again...

DinoSlavik committed:
Added some translations

SuperMaxusa commented on src/main/resources/assets/opencomputers/doc/uk_UA/block/assembler.md:
```diff
@@ -0,0 +1,13 @@
+# Конструктор Електротехніки
```
Maybe the word "[складальник](http://sum.in.ua/s/skladaljnyk)" would be more appropriate here?

DinoSlavik committed:
Update assembler.md

SuperMaxusa commented on src/main/resources/assets/opencomputers/doc/uk_UA/block/assembler.md:
```diff
@@ -0,0 +1,13 @@
+# Складальник Електротехніки
+
+![Місніше, краще, швидше, сильніше.](oredict:oc:assembler)
```
```suggestion
![Harder, better, faster, stronger.](oredict:oc:assembler)
```
Це відсилка до пісні "Harder, Better, Faster, Stronger", тому такі речі просять перекладати не дослівно: https://github.com/MightyPirates/OpenComputers/pull/2091#discussion_r82571005

SuperMaxusa commented on src/main/resources/assets/opencomputers/doc/uk_UA/block/index.md:
```diff
@@ -0,0 +1,48 @@
+## Комп'ютери
+* [Комп'ютерний Корпус](case1.md)
+* [Мікроконтролер](microcontroller.md)
+* [Серверна Стійка](rack.md)
```
```suggestion
* [RAID](rack.md)
```
Поки не знаю, що тут краще підходить: просто абревіатура "[RAID](https://uk.wikipedia.org/wiki/RAID)", "RAID-масив" або "[Дисковий масив](https://uk.wikipedia.org/wiki/%D0%94%D0%B8%D1%81%D0%BA%D0%BE%D0%B2%D0%B8%D0%B9_%D0%BC%D0%B0%D1%81%D0%B8%D0%B2)"

DinoSlavik committed:
Update src/main/resources/assets/opencomputers/doc/uk_UA/block/assembler.md

Co-authored-by: SuperMaxusa <41739128+SuperMaxusa@users.noreply.github.com>

DinoSlavik committed:
Виправлено трішки маленьких помилок.

DinoSlavik committed:
Update uk_UA.lang

DinoSlavik committed:
Перекладено домашню сторінку предметів

Переклав домашню сторінку предметів.
Стандартизував стиль написання назв, що містять "для".
Прибрав деякі неважливі скорочення у назвах для поліпшення читабельності.
Змінив наступні назви:
  - "Стрілочки" на "Стрілки"
  - "Покращений Інвентар" на "Інвентар" (за прикладом "Резеверуару")
  - "Покращення Повідок" на "Ласо"
  - "Покращений Магніт" на "Магніт

DinoSlavik committed:
Маленькі виправлення
